mu4e/mu4e-compose.el

```diff
@@ -146,7 +146,7 @@ Also see `mu4e-context-policy'."
 	   :group 'mu4e-compose))
 
 
-(defcustom mu4e-compose-crypto-reply-policy 'sign-and-encrypt
+(defcustom mu4e-compose-crypto-reply-encrypted-policy 'sign-and-encrypt
   "Policy for signing/encrypting replies to encrypted messages.
 We have the following choices:
 
```

Comment:
I think we'd need a `make-obsolete-variable`, to warn users of the old variable.

Reply:
I'm not sure about the standard way of doing that. I proposed something in commit  e761f74.